apps/demo/views/layout.html

```diff
@@ -63,12 +66,12 @@
   {{$footerSupportLinks}}
     <ul>
       {{#footerSupportLinks}}
-        <li><a href="{{path}}">{{#t}}{{property}}{{/t}}</a></li>
+        <li class="govuk-footer__inline-list-item"><a href="{{path}}">{{#t}}{{property}}{{/t}}</a></li>
       {{/footerSupportLinks}}
       {{^footerSupportLinks}}
-        <li><a href="/cookies">{{#t}}base.cookies{{/t}}</a></li>
-        <li><a href="/terms-and-conditions">{{#t}}base.terms{{/t}}</a></li>
-        <li><a href="/accessibility">{{#t}}base.accessibility{{/t}}</a></li>
+        <li class="govuk-footer__inline-list-item"><a class="govuk-footer__link" href="/cookies">{{#t}}base.cookies{{/t}}</a></li>
+        <li class="govuk-footer__inline-list-item"><a  class="govuk-footer__link" href="/accessibility">{{#t}}base.accessibility{{/t}}</a></li>
+        <li class="govuk-footer__inline-list-item"><a  class="govuk-footer__link" href="/terms-and-conditions">{{#t}}base.terms{{/t}}</a></li>
       {{/footerSupportLinks}}
     </ul>
   {{/footerSupportLinks}}
```

Comment:
is there a reason to override the main layout in hof? I think you can change the footer links dynamically, I think it's done in my pr for end of tenancy

Reply:
the reason the layout is overriding is because of the nav bar not the footer -  the change I made to the footer links makes them the same as the main layout in hof. We could change the nav bar to it's own page with the options (similar to the sandbox) and then we won't need to override the main layout and can remove that view from the demo app?

Reply:
separate nav bar page sounds good 👍